csharp/IceGrid/IceBox/simpsons.xml

```diff
@@ -0,0 +1,42 @@
+<icegrid>
+  <!-- An application describes nodes and servers deployed on these nodes. An IceGrid deployment (a registry plus one or
+  more nodes) can host several applications. -->
+  <application name="Simpsons">
+
+    <!-- A service template allows us to instantiate several similar IceBox services. -->
+    <service-template id="GreeterService">
+      <parameter name="name"/>
+        <service name="${name}" entry="Service/bin/Debug/net8.0/GreeterService.dll:Service.GreeterService">
+            <description>Get a greeting from ${name}!</description>
+
+            <!-- The object adapter created by this service. It listens on an OS-assigned tcp port, on all interfaces.
+            It also defines a well-known object with identity "greeter-${name}". IceGrid also generates property
+            Greeter.Identity in the config file for the service instance.
+            server-lifetime="false" tells IceGrid this object adapter has not the same lifetime as the IceBox server;
+            this is the recommended setting for object adapters in IceBox services since an IceBox service can be
+            started, then stopped, then restarted again while the IceBox server keeps running. -->
+            <adapter name="GreeterAdapter" endpoints="tcp" server-lifetime="false">
+              <object identity="greeter-${name}" type="::VisitorCenter::Greeter" property="Greeter.Identity"/>
+            </adapter>
+
+            <!-- Generate the Greeter.Name property. -->
+            <property name="Greeter.Name" value="${name}" />
+         </service>
+    </service-template>
+
+    <!-- Describes the servers deployed on the IceGrid node named 'node1'. -->
+    <node name="node1">
+
+      <!-- Describes the server deployed on node1. IceGrid starts this server when a client looks up an object adapter
+      or a well-known object in this server. Here, we deploy a single IceBox server that loads 4 IceBox services. -->
+      <icebox id="IceBox" activation="on-demand" exe="dotnet">
```

Comment:
The previous demo (old-icebox) deployed the services as replicas, which I don't think is appropriate here. Each service produces a distinct greeting ... they are not replicas.